plugins/ssr/ssrvernac.mlg

```diff
@@ -309,9 +309,12 @@ END
       ~category:"deprecated" ~default:CWarnings.Enabled
       (fun () ->
          (Pp.strbrk
-            "SSReflect's Search command has been moved to the \
-             ssrsearch module; please Require that module if you \
-             still want to use SSReflect's Search command"))
+            "The Search command has been changed to refer to Coq's own \
+             search feature (formerly SearchAbout).\
+             In previous versions of Coq, the Search command was provided \
```

Comment:
```suggestion
             In previous versions of Coq, the Search command was overridden \
```

Reply:
Was that previous `Search` equivalent to `SearchAbout`?

Reply:
Yes.

Reply:
Okay so to make sure I understand: There are two search command, Coq's and ssreflect's.
Previously, without loading ssreflect, `Search` and `SearchAbout` would both refer to Coq's.
When loading ssreflect, `Search` was changed to the ssreflect version, but `SearchAbout` was left unchanged.

Now `SearchAbout` is deprecated, and morevoer loading ssreflect does *not* override `Search` any more. But one can still opt-in to the overriding by `Require ssrsearch`.

Is that correct?

Reply:
I am surprised that `Search` and `SearchAbout` would have been equivalent without importing ssreflect, that's new to me (why do both of these names even exist?)... but I guess I never read the docs carefully enough.^^

Reply:
Ah, the docs say there also used to be `SearchHead` but that is some long-dead history.

Reply:
> Is that correct?

That's fully correct.

>why do both of these names even exist?

Historical reasons. They used to mean something different but that was changed a long time ago (in 8.4 or 8.5 I believe).